bokeh/models/sources.py

```diff
@@ -210,6 +210,18 @@ class RemoteSource(DataSource):
 class AjaxDataSource(RemoteSource):
     method = String('POST', help="http method - GET or POST")
 
+    overwrite = Bool(True, help="""
+    Overwrite or Update plot values from data source object
+    """)
+    max_size = Int(help="""
```

Comment:
no need to reference plots here, this is purely about the data source buffer size